src/components/logs/forms/medication-upload.tsx

```
@@ -0,0 +1,189 @@
+            <div className=' flex items-start justify-center gap-4 max-w-lg text-center'>
+                <Info className='size-5' />
+                No medication setup found. Please create a medication setup first in the setup tab.
+            </div>
+        </div>
+    }
+
+    const medicationId = useStore(form.store, (state) => state.values.medicationId)
+
+    useEffect(() => {
+        if (medicationId === 0) {
+            form.setFieldValue('medicationId', medicationSetup?.[0]?.id!)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)